jams/schemata/namespaces/pitch/note_midi.json

```diff
@@ -0,0 +1,8 @@
+{"note_midi":
+    {
+        "value": { "type": "number" },
```

Comment:
Yes -- it's more of a question of how far we want to (pitch) bend the rules of MIDI.  Are there accepted practices for that kind of thing?